docker/db.Dockerfile

```diff
@@ -0,0 +1,9 @@
+from postgres:13.4
+
+RUN echo "\connect postgres" >> /docker-entrypoint-initdb.d/init.sql
```

Comment:
it is better to add the wal commands to a `priv/wal.sql` and mount it to the postgres image's initdb volume. That way we don't have to build the db image and have one file to handle.

Something like:
```
db:
    volumes:
      - /priv/wal.sql:/docker-entrypoint-initdb.d/wal.sql
      - pg-data:/var/lib/postgresql/data
```

Reply:
Mounting sql files to the init directory is in the first line of the section you're referring to:
<img width="780" alt="Screenshot 2022-12-16 at 1 10 54 PM" src="https://user-images.githubusercontent.com/22714384/208027216-b6ff156d-5066-4ebb-86df-0eabea32b347.png">

Reply:
ok that was dumb on my part, will tackle it the way you mentioned